### ISSUE TITLE
Signs not showing up

### ISSUE THREAD

**Issue by gocke004** [mapcrafter/mapcrafter#302]
Dear mapcrafter community,

Let me just say that I am very excited and thankful to see that you guys have made mapcrafter 1.13-compatible. It seems that this update has stunted similar mapping programs like overviewer or tectonicus. Thank you for all the hard work!

I am relatively new to mapcrafter. I am having trouble getting the program to render signs in my worlds. Could someone help me figure out how to get signs to show up? I can't figure out where I've gone wrong...I have based my configuration file on some of those which have been posted by ligos at https://minecraft.ligos.net/worlds/index.html.

Here is some further information:
**mapcrafter version:** 2.3.1-206-gfc9ea96_win64 (one of the nightly builds)
**Minecraft version:** 1.13
**Configuration File:**

> output_dir =  C:\Users\Sam\Desktop\mapcrafter_2.3.1-206-gfc9ea96_win64\Output\4
> 
> [world:myworld]
> input_dir = C:\Users\Sam\Desktop\mapcrafter_2.3.1-206-gfc9ea96_win64\worlds\world
> 
> [global:map]
> world = myworld
> render_view = isometric
> render_mode = daylight
> rotations = top-left
> texture_size = 12
> 
> [map:map_myworld]
> name = My World
> world = myworld
> 
> [marker:all_signs]
> name = Signs
> show_default = false
> title_format = %(text)
> 
> 

The map renders just fine and I can see all of my chunks properly loaded, but I just can't see my signs. I'm sure I must be doing something wrong...
![untitled](https://user-images.githubusercontent.com/44779311/48308364-8190c080-e528-11e8-834c-af2aaa164daf.png)

Thank you for your help, and thank you for keeping mapcrafter alive!
Sam

**Comment by ligos** [comment]
Hi Sam. Glad someone is using the example configs I made!

Just double checking if you're running the `mapcrafter_markers.exe` program? Mapcrafter itself doesn't do anything with the `[marker:...]` sections. If all goes well you should end up with a `markers-generated.js` file with the positions of your signs, but only after you run `mapcrafter_markers`.

I think some changes were recently merged into the doco which might help: https://docs.mapcrafter.org/builds/world113/markers.html

**Comment by gocke004** [comment]
@ligos Thank you! It is now working!! I have my signs rendered and everything. It was very fast, too. I did not know that the mapcrafter_markers.exe had to be run separately but that makes good sense.

Thank you very much for your help. I am very glad that there are still people working on this very powerful little app. Have a nice day.
-Sam
